src/server/auth/cmds/cmds_integration_test.go

```diff
@@ -94,7 +95,7 @@ func TestLogin(t *testing.T) {
 		var buf bytes.Buffer
 		cmd.Stderr = &buf
 
-		c = tu.UnauthenticatedPachClient(t, c)
+		c := tu.UnauthenticatedPachClient(t, c)
```

Comment:
This is probably the right thing to do; guessing this test never retried successfully.